creator-node/src/routes/tracks.js

```diff
@@ -314,6 +322,7 @@ module.exports = function (app) {
       // Error on outdated blocknumber
       const cnodeUser = req.session.cnodeUser
       if (blockNumber < cnodeUser.latestBlockNumber) {
+        metric.saveTime({ scope: 'error' })
```

Comment:
doesn't saveTime require a startTime param as well?

Reply:
i see what's happening. PrometheusMetric is a new instance of the class that gets initialized every time and the constructor calls resetTimer which sets startTime. i do agree with sid here though, i don't think users will always necessary be aware of the exact location in code when they init this class, might be safer to expect a startTime to be passed in